lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -2385,6 +2385,23 @@ SPIRVEntry *parseAndCreateSPIRVEntry(SPIRVWord &WordCount, Op &OpCode,
   if (WordCount == 0 || OpCode == OpNop) {
     return nullptr;
   }
+  if (!SPIRVUseTextFormat) {
```

Comment:
I though this should be fixing clang-tidy error, isn't it the case?
```
static void validateWordCount(SPIRVModuleImpl &M, std::istream &IS,
                              SPIRVWord WordCount) {
/* */
}
```

Reply:
I've opened https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/3286 to address this.